### ISSUE TITLE
MuonClip lr_muon=0.01 causes training divergence with BitNet

### ISSUE THREAD

**Issue by Lev-Stambler** [DeepOpt-com/WrinkleFree-Monorepo#25]
## Problem

When using MuonClip optimizer with `lr_muon=0.01` (the default in muon-clip examples), BitNet training diverges catastrophically.

## Ablation Results (2026-01-02)

| Optimizer | Step 16 Loss | Step 500+ Loss | Perplexity | Status |
|-----------|-------------|----------------|------------|--------|
| MuonClip (`lr_muon=0.01`) | 24.0 | 32.4 | 2.4B | ❌ Diverged |
| AdamW (`lr=1e-4`) | 2.4 | 7.7 | 168 | ✅ Working |

## Root Cause

The `lr_muon=0.01` learning rate, which works well for full-precision models, is too aggressive for BitNet's ternary weight training. The STE (Straight-Through Estimator) gradients combined with high LR cause rapid divergence.

## Current Workaround

Switched default optimizer to AdamW in `lrc_dlm_influence.yaml`:

```yaml
optimizer:
  type: adamw
  lr_adam: 1e-4
  weight_decay: 0.01
```

## Future Work

- [ ] Tune MuonClip LR for BitNet (try `lr_muon=0.001` or `lr_muon=0.0005`)
- [ ] Consider warmup schedule for Muon LR during lambda warmup phase
- [ ] Investigate if momentum needs adjustment for quantized training

## WandB Runs

- MuonClip (diverged): https://wandb.ai/umd-leans-well/wrinklefree/runs/ablation_v1_muonclip-6d655581
- AdamW (working): https://wandb.ai/umd-leans-well/wrinklefree/runs/ablation_v1_adamw-74ac9fa5

## Related

- MuonClip repo: https://github.com/GAD-cell/muon-clip
- BitNet paper: https://arxiv.org/abs/2402.17764
